src/routes/request.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import RequestController from '../controllers/requestController';
+
+const router = express.Router();
+
+router.post('/multi-city', RequestController.multiCityRequest);
```

Comment:
Please apply destructuring in your code. 

Example
Instead of having `RequestController.multiCityRequest`

you can destructure it up by:
```
const { multiCityRequest } = RequestController;

router.post('/multi-city', multiCityRequest);

```